### PR TITLE
✨ [Portfolio] Show user statistical data

### PR DESCRIPTION
**Pull request by WeiJun0827**

### CODE REVIEW

File: src/components/NFTPortfolio/UserInfo.vue [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    </Label>
+    <div v-if="likerDescription" :class="['w-full', 'h-[1px]', 'bg-shade-gray', 'my-[16px]']" />
+    <Label preset="p6" class="break-all font-200">
+      {{ likerDescription }}
+    </Label>
+    <div :class="['w-full', 'h-[1px]', 'bg-shade-gray', 'my-[16px]']" />
+    <Label preset="p6" class="font-200">Collections: {{ collectedNFTs.length }}</Label>
+    <Label preset="p6" class="font-200">TotalValue: {{ collectedNFTTotalValue }}</Label>
+    <Label preset="p6" class="font-200">Creations: {{ createdNFTClassIds.length }}</Label>
+    <Label preset="p6" class="font-200">Minted: {{ createdNFTMintedCount }}</Label>
```
Comment (nwingt) [inline src/components/NFTPortfolio/UserInfo.vue:15]: Can wrap the 4 Label into another component e.g. `NFTPortoflio/Stats.vue`

**Review by nwingt — CHANGES_REQUESTED** [review]

File: src/components/NFTPortfolio/State.vue [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      return this.collectedNFTs.reduce(
+        (total, nft) =>
+          total + (this.getNFTClassPurchaseInfoById(nft.classId)?.price || 0),
+        0
+      );
+    },
+    createdNFTClassIdCount() {
+      return this.createdNFTClassIds.length;
+    },
+    createdNFTMintedCount() {
```
Comment (nwingt) [inline src/components/NFTPortfolio/State.vue:40]:
```suggestion
    createdCollectedCount() {
```

File: src/components/NFTPortfolio/UserInfo.vue [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    <div v-if="likerDescription" :class="['w-full', 'h-[1px]', 'bg-shade-gray', 'my-[16px]']" />
+    <Label preset="p6" class="break-all font-200">
+      {{ likerDescription }}
+    </Label>
+    <div :class="['w-full', 'h-[1px]', 'bg-shade-gray', 'my-[16px]']" />
+    <NFTPortfolioState 
+      :collected-n-f-ts="collectedNFTs"
+      :created-n-f-t-class-ids="createdNFTClassIds"
+    >
+      <template v-slot="slotProps">
```
Comment (nwingt) [inline src/components/NFTPortfolio/UserInfo.vue:16]:
```suggestion
      <template v-slot="stats">
```

File: src/components/NFTPortfolio/UserInfo.vue [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    },
+    wallet: {
+      type: String,
+      default: null,
+    },
+    collectedNFTs: {
+      type: Array,
+      default: () => [],
+    },
+    createdNFTClassIds: {
```
Comment (nwingt) [inline src/components/NFTPortfolio/UserInfo.vue:45]:
```suggestion
    createdClassIds: {
```

File: src/components/NFTPortfolio/UserInfo.vue [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  props: {
+    userInfo: {
+      type: Object,
+      default: null,
+    },
+    wallet: {
+      type: String,
+      default: null,
+    },
+    collectedNFTs: {
```
Comment (nwingt) [inline src/components/NFTPortfolio/UserInfo.vue:41]:
```suggestion
    collectedItems: {
```

File: src/components/NFTPortfolio/State.vue [hunk 6]
```diff
@@ -0,0 +1,48 @@
+<template>
+  <div>
```
Comment (nwingt) [inline src/components/NFTPortfolio/State.vue:2]: Can use `<component>` with `tag` prop with default is `div`
```suggestion
  <component :is="tag">
```

File: src/components/NFTPortfolio/State.vue [hunk 7]
```diff
@@ -1,12 +1,12 @@
 <template>
-  <div>
+  <component :is="tag">
```
Comment (nwingt) [inline src/components/NFTPortfolio/State.vue:2]: need to add `tag` to props

Comment (WeiJun0827) [reply src/components/NFTPortfolio/State.vue:2]: Seems I added changes in the wrong [commit](https://github.com/likecoin/liker-land/pull/556/commits/471e0f90efa65932d9ac71252f8c35521189d21b#diff-e360e555023d5c11dcb5a54cdab861b6ab9aafc4f54de2fa1b953463c44356edR16-R19) 😅
Sorry for being confusing.
I will resubmit commits.

**Review by nwingt — APPROVED** [review]
Please squash away the second to the last commit

**Pull request merged**